.github/workflows/build.yml

```diff
@@ -472,7 +472,7 @@ jobs:
         run: |
             sudo pkg update
             sudo pkg install -y gmake automake autoconf pkgconf llvm15 clinfo clover opencl clblast openblas
-            gmake CC=/usr/local/bin/clang15 CXX=/usr/local/bin/clang++15
+            gmake CC=/usr/local/bin/clang15 CXX=/usr/local/bin/clang++15 -j `sysctl -n hw.ncpu`
```

Comment:
any reason you switched syntax for this one?

Reply:
The FreeBSD shell doesn't support the `$()` syntax.

Reply:
I see. Should we switch all the others then?
This is probably the difference between `sh` and `bash`

Reply:
> This is probably the difference between `sh` and `bash`

Nope. Modern POSIX sh supports `$()`, you can try it out for yourself with the dash shell. FreeBSD defaults to tcsh which doesn't support `$()`.